substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -425,8 +425,8 @@ public Path cachePath() {
         }
     }
 
-    private static Path fullFilePathFromClassName(HostedInstanceClass hostedInstanceClass) {
-        String[] elements = hostedInstanceClass.toJavaName().split("\\.");
+    private static Path fullFilePathFromClassName(HostedType hostedClass) {
+        String[] elements = hostedClass.toJavaName().split("\\.");
```

Comment:
It's not clear to me how this change relates to making the SourceManager thread safe, could you please explain?